cmd/humioctl/packages_install.go

```diff
@@ -68,3 +92,43 @@ func isDirectory(path string) (bool, error) {
 	}
 	return fileInfo.IsDir(), err
 }
+
+func getURLPackage(url string) (*os.File, error) {
+	// Github URLs should have the .got suffix removed
+	if strings.HasPrefix(url, "https://github.com") || strings.HasPrefix(url, "http://github.com") {
+		url = strings.TrimSuffix(url, ".git")
+	}
+
+	zipBallURL := url + "/zipball/master/"
+	response, err := http.Get(zipBallURL)
+
+	if err != nil {
+		return nil, err
+	}
+
+	if response.StatusCode >= 400 {
+		return nil, fmt.Errorf("error downloading file %s: %s", zipBallURL, response.Status)
+	}
+
+	defer response.Body.Close()
+	zipContent, err := ioutil.ReadAll(response.Body)
+
+	if err != nil {
+		return nil, err
+	}
+
+	tempDir := os.TempDir()
+	zipFile, err := ioutil.TempFile(tempDir, "humio-package.*.zip")
+
+	if err != nil {
+		return nil, err
+	}
+
+	_, err = zipFile.Write(zipContent)
```

Comment:
you can do `io.Copy(zipFile, response.Body)`, to avoid buffering in memory.

Reply:
Thanks. I will get that updated.